base-notebook/docker_healthcheck.py

```diff
@@ -16,6 +16,11 @@
 url = json.loads(json_file.read_bytes())["url"]
 url = url + "api"
 
-r = requests.get(url, verify=False)  # request without SSL verification
+proxies = {
+    "http": None,
+    "https": None,
+}
```

Comment:
```suggestion
proxies = {
    "http": "",
    "https": "",
}
```

This should solve mypy issue.
There seems to be some difference  though.
https://stackoverflow.com/a/72529216/4881441